frontend/src/photoView/photoView.js

```diff
@@ -2,14 +2,16 @@ import React from 'react';
 import Navbar from '../about/Navbar';
 import { Footer } from '../UILibrary/components';
 
+const SIDES = ['front', 'back', 'binder'];
```

Comment:
Ideally what we'd like to do for a situation like this is use an enum, which JavaScript doesn't really natively support (one of the many reasons to love [TypeScript](https://www.typescriptlang.org/docs/handbook/enums.html)) .

Here's the workaround -- use an object. So:
```
const SIDES = {
    FRONT: 'front',
    BACK: 'back',
    BINDER: 'binder',
}
```

Then, when later you're switching on which side, instead of switching on a string like this: `if (this.state.display_side === 'front')`

you can do `if (this.state.display_side === SIDES.FRONT)`

Since you're no longer using a string literal but instead a variable, this is now much more robust to things like typos.